drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -884,31 +874,33 @@ def read_hlet_wcs(filename, ext):
     """
     hstwcs = wcsutil.HSTWCS(filename, ext=ext)
     if hstwcs.naxis1 is None:
```

Comment:
@nden Is there any circumstance under which `naxis` could be `None` in a WCS?

Reply:
Only if you create a default WCS without passing a data file.
Unless `filename` above can be `None`, `naxis1/2` should always be populated.

Reply:
@nden Thanks! I was just curious (+wanted to e sure)

Reply:
@nden Then maybe this `if` can be removed?

Reply:
This PR is simply PEP8 changes. I'll try to review the code today and will include it there.